stats/src/test/java/io/airlift/stats/TestQuantileDigest.java

```diff
@@ -780,4 +838,16 @@ private void addRange(QuantileDigest digest, int from, int to)
         }
         digest.validate();
     }
+
+    private static long doubleToSortableLong(double value)
+    {
+        long bits = Double.doubleToLongBits(value);
+        return bits ^ (bits >> 63) & Long.MAX_VALUE;
```

Comment:
Add clarifying parenthesis. It's easy to forget the precedence rules for `^` and `&` so that will make the code easier to reason about.